### DISCUSSION TITLE
How do I set the window title to something other than "Bevy App"?

### DISCUSSION THREAD

**Discussion by Newbytee** [bevyengine/bevy#8632] [Q&A] [answered]
![image](https://github.com/bevyengine/bevy/assets/4428753/02ed9ae1-d53f-4b81-861e-aa826c3d2503)

By default, the window title in Bevy is set to "Bevy App". How can I change this to something else?

**Comment by ickshonpe** [comment, marked as answer]
Two methods:
* The `DefaultPlugins::set` function: 
    ```rust
    fn main() {
        App::new()
            .add_plugins(DefaultPlugins
                .set(WindowPlugin {
                    primary_window: Some(Window {
                        title: "something other than \"Bevy App\"".to_string(),
                        ..Default::default()
                    }),
                    ..Default::default()
                })
            )
            // .. the rest of your app
            .run();
    }
    ```

* Query for `Window`  from a system:
    ```rust
    fn set_window_title(
        mut window_query: Query<&mut Window, With<PrimaryWindow>>,
    ) {
        if let Ok(window) = window_query.get_single_mut() {
            window.title = "something other than \"Bevy App\"".to_string();
        } 
    }
    ```

**Comment by ickshonpe** [comment]
Also, you might find the [window](https://github.com/bevyengine/bevy/tree/main/examples/window) examples helpful. In particular [multiple_windows](https://github.com/bevyengine/bevy/blob/main/examples/window/multiple_windows.rs) and [window_settings](https://github.com/bevyengine/bevy/blob/main/examples/window/window_settings.rs).
